eland/common.py

```diff
@@ -26,12 +26,12 @@
 
 # Default number of rows displayed (different to pandas where ALL could be displayed)
 DEFAULT_NUM_ROWS_DISPLAYED = 60
-
 DEFAULT_CHUNK_SIZE = 10000
 DEFAULT_CSV_BATCH_OUTPUT_SIZE = 10000
 DEFAULT_PROGRESS_REPORTING_NUM_ROWS = 10000
 DEFAULT_ES_MAX_RESULT_WINDOW = 10000  # index.max_result_window
 DEFAULT_PAGINATION_SIZE = 5000  # for composite aggregations
+PANDAS_VERSION = pd.__version__
```

Comment:
We would prefer a tuple of ints for easier/correct comparisons:

`PANDAS_VERSION: Tuple[int, int] = tuple(int(part) for part in pd_version.split(".") if part.isdigit())[:2]`